composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "guzzlehttp/guzzle": "~5.0|~6.0",
         "components/handlebars.js": "~3.0.0",
-        "symfony/symfony": "^2.7.7"
+        "symfony/symfony": "^3.4@beta"
```

Comment:
should it be ´^2.7.7|^3.3` ? 

1. if root asks for 3.4@beta it will pass the requirement without having to set beta here
2. as 2.0.0 has bee not since august, unless we have a need for only making it available to v2 users, maybe we can avoid that :)

Reply:
I also prefer soft requirement. Backported it to 1.2 in #100 because of php-cs-fixer dependencies, so that would solve this PR as well.